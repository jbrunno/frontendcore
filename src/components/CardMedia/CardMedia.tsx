import React, { ElementType } from 'react';
import {
  CardMedia as MuiCardMedia,
  CardMediaProps as MuiCardMediaProps,
} from '@mui/material';

export type CardMediaProps = MuiCardMediaProps & {
  component?: ElementType;
};

export function CardMedia(props: CardMediaProps) {
  return <MuiCardMedia {...props} />;
}

export default CardMedia;
