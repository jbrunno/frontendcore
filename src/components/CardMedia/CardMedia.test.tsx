import React from 'react';
import { render } from '@test-utils/component-testing';
import { CardMedia } from './CardMedia';

test('renders the CardMedia', () => {
  const { container } = render(<CardMedia />);
  expect(container).toMatchSnapshot();
});
