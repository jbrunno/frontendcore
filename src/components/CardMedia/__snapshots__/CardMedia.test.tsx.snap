// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the CardMedia 1`] = `
.emotion-0 {
  display: block;
  -webkit-background-size: cover;
  background-size: cover;
  background-repeat: no-repeat;
  -webkit-background-position: center;
  background-position: center;
}

<div>
  <div
    class="MuiCardMedia-root emotion-0"
  />
</div>
`;
